build/saucelabs-unit-test.js

```diff
@@ -70,7 +70,13 @@ const waitingCallback = (error, body, id) => {
       // Exit
       if (jobsDone === browsersFile.length - 1) {
         jsUnitSaucelabs.stop()
-        process.exit(jobsDone === jobsSucceeded ? 0 : 1)
+        if (jobsDone > jobsSucceeded) {
```

Comment:
Any reason not to use `!==`?

Reply:
Nope but for me it's more clear than using `!==` here because the line after we substract those two numbers